arches/management/commands/packages.py

```diff
@@ -343,16 +349,35 @@ def handle(self, *args, **options):
             self.create_mapping_file(options["dest_dir"], options["graphs"])
 
         if options["operation"] in ["load", "load_package"]:
-            defer_indexing = False if str(options["defer_indexing"])[0].lower() == "f" else True
+            arches_application = options['arches_application']
+            arches_application_path = None
+
+            if arches_application:
+                site_package_path = site.getsitepackages()[0]
+                arches_application_path = os.path.join(site_package_path, arches_application)
+
+                if not os.path.exists(arches_application_path):
+                    egg_link_path = arches_application_path.replace('_', '-')
+                    egg_link_path += '.egg-link'
+
+                    if os.path.exists(egg_link_path):
+                        original_path = Path(egg_link_path).read_text()
+                        original_path = original_path.replace('.', '')
+                        original_path = original_path.strip()
+
+                        arches_application_path = os.path.join(original_path, arches_application)
+
+                arches_application_path = os.path.join(arches_application_path, 'pkg')
+
             self.load_package(
```

Comment:
Going to back to regular pip installs without `-e`, I found that I could not run the `load_package -a {INSTALLED_APP}` successfully, because the `load_package()` method looks at `settings.APP_ROOT`, in order to find all the goodies, but that doesn't work when APP_ROOT is the name of the consuming application, like `jacobs-bespoke-app` generated with `arches-project create` which doesn't have all the goodies. Or am I missing something important about how this works with APP_ROOT?

Reply:
should be resolved in latest